### ISSUE TITLE
[Swift] Codegen does not detect file data types

### ISSUE THREAD

**Issue by AnthonyOliveri** [swagger-api/swagger-codegen#4764]
##### Description

For Swift codegen, the `CodegenOperation`'s `defaultResponse` is never set to type `file`, even when specified in the swagger doc, like so:

```yaml
responses:
    200:
        description: success
        schema:
            type: file
```

This means that anything nested in the mustache tags `{{#operation}}{{#isResponseFile}}` never get evaluated, even for file type responses.
 

##### Swagger-codegen version

master branch

##### Suggest a Fix

The fix is very simple: the function `public boolean isDataTypeFile(String dataType)` needs to be implemented in `SwiftCodegen.java` and `Swift3Codegen.java`. Since these methods are currently not implemented, the response type incorrectly uses the same function from `DefaultCodegen.java`, which looks for the type string "file" instead of the Swift equivalent "NSURL" or "URL".

I can create a pull request with the fix if you like.

**Comment by wing328** [comment]
@AnthonyOliveri thanks for reporting the issue. Please submit a PR so that we can review the change.

**Comment by AnthonyOliveri** [comment]
Submitted pull request #4787

**Comment by wing328** [comment]
Closed via #5056 

Thanks for the contribution.
